keyboards/splitkb/aurora/corne/keymaps/vial/rules.mk

```
@@ -0,0 +1,17 @@
+# Copyright 2022 splitkb.com <support@splitkb.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
```

I'd say that you could delete this file entirely but you probably want to start selectively disabling features here so the firmware actually builds. Here's an example of that syntax.
```suggestion
ifeq ($(strip $(MCU)), atmega32u4)
    QMK_SETTINGS = no
endif
```
...Also you can leave the huge license blob off `rules.mk` files. QMK's collaborators [have actually been trying to get splitkb's contractors to stop cut-pasting it in](https://github.com/qmk/qmk_firmware/pull/21499#issuecomment-1632697365).